docs/site/content/docs/getting-started/add-to-existing-repository.mdx

```diff
@@ -197,6 +198,86 @@ Turborepo optimizes your repository using information from your package manager.
   yet.
 </Callout>
 
+</Step>
+<Step>
+### Set up package manager workspaces
+
+For multi-package repositories, you'll need to configure your package manager to recognize your workspace structure. This is done by adding a `workspaces` field to your root `package.json`.
```

Comment:
Let's make "multi-package repositories" a link to https://vercel.com/docs/glossary#multi-package-workspace and have the copy be "multi-package workspaces".